### PR TITLE
Modify GenotypeConcordance to explicitly require that the VCF files be indexed if an interval file is specified

### PR DESCRIPTION
**Pull request by gbggrant**
Modify GenotypeConcordance to explicitly require that the VCF files be indexed if an interval file is specified.  Otherwise it would blow up, somewhat more obscurely, for lack of an index file.

### CODE REVIEW

**Comment by vdauwera** [comment]
Assuming @bradtaylor doesn't mind being unassgined, this sounds like something that could be done by one of our consultants.

**Comment by gbggrant** [comment]
I think all that's left is a final ok.  Kind of fell through the cracks.

**Comment by vdauwera** [comment]
@bradtaylor @gbggrant Sorry for bugging you guys with this, but might this final ok be forthcoming? Would like to clean up the issue tracker, plug the cracks, etc.

File: src/java/picard/vcf/GenotypeConcordance.java [hunk 1]
```diff
@@ -138,7 +138,8 @@ public static void main(final String[] args) {
         IntervalList intervals = null;
         SAMSequenceDictionary intervalsSamSequenceDictionary = null;
         if (usingIntervals) {
-            log.info("Loading up region lists.");
+            USE_VCF_INDEX = true;   // If intervals are specified, the VCF files must be indexed
```
Comment (nh13) [inline src/java/picard/vcf/GenotypeConcordance.java]: I would suggest put this in the custom command line validation or emit a log warning stating you are setting ```UCE_VCF_INDEX```.

**Comment by nh13** [comment]
@vdauwera I added a comment here.  @bradtaylor @gbggrant should agree on the final solution before merging.

**Comment by bradtaylor** [comment]
I like the custom command line validation. Thumbs up!

**Pull request merged**